kinds/dashboard/dashboard_kind.cue

```diff
@@ -124,16 +132,16 @@ lineage: schemas: [{
 		#AnnotationTarget: {
 			// Only required/valid for the grafana datasource...
 			// but code+tests is already depending on it so hard to change
-			limit: int64
+			limit?: int64
 			// Only required/valid for the grafana datasource...
 			// but code+tests is already depending on it so hard to change
-			matchAny: bool
+			matchAny: bool | *false
 			// Only required/valid for the grafana datasource...
 			// but code+tests is already depending on it so hard to change
-			tags: [...string]
+			tags?: [...string]
 			// Only required/valid for the grafana datasource...
 			// but code+tests is already depending on it so hard to change
-			type: string
+			type?: string
```

Comment:
i made these changes because it caused a ton of dashboards to fail validation, and because the type just wasn't in line with the comment (if it's only required _sometimes_, then it should be optional in general)

There's a way of writing this where we can capture the conditional possibility described in the comments, but i didn't want to go down this rabbit hole.

An alternative would be to roll these back and just add more dashboards to the skiplist.

cc @ivanortegaalba